common/changes/@rushstack/eslint-patch/ating-eslint-sarif-formatter_2024-10-02-21-04.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/eslint-patch",
+      "comment": "Upgraded @types/eslint to version 8.56.10",
```

```suggestion
      "comment": "",
```

The comment should be cleared out too.